packages/conf/src/getConfig.ts

```diff
@@ -19,6 +20,29 @@ function JitiLoader(): LoaderSync {
   }
 }
 
+const moduleName = "lingui"
+
+const configExplorer = cosmiconfigSync(moduleName, {
+  searchPlaces: [
+    `${moduleName}.config.js`,
+    `${moduleName}.config.cjs`,
+    `${moduleName}.config.ts`,
+    `${moduleName}.config.mjs`,
+    "package.json",
+    `.${moduleName}rc`,
+    `.${moduleName}rc.json`,
+    `.${moduleName}rc.yaml`,
+    `.${moduleName}rc.yml`,
+    `.${moduleName}rc.ts`,
+    `.${moduleName}rc.js`,
+  ],
+  loaders: {
+    ".js": JitiLoader(),
+    ".ts": JitiLoader(),
+    ".mjs": JitiLoader(),
+  },
+})
```

Comment:
i read in the cosmiconfig docs that hey have a cache on the configExplorer instacne. Pulling creating this explorer out of the `getConfig` function should help to leverage this cache.